### PR TITLE
Ensure the message can be processed as text

### PR DESCRIPTION
**Pull request by eduardoj**
Same as yast/yast-journal#21

### CODE REVIEW

File: src/lib/journalctl/entry_presenter.rb [hunk 1]
```diff
@@ -50,5 +50,10 @@ def source
     def formatted_time
       timestamp.strftime(TIME_FORMAT)
     end
+
+    # Message string
+    def message
+      entry.message.to_s
```
Comment (jreidinger) [inline src/lib/journalctl/entry_presenter.rb]: please add also explanation why it is needed. Especially as this is tutorial module, it need explanation why it is needed this way.

**Comment by lslezak** [comment]
Ping, @eduardoj I guess adding a similar comment as in yast/yast-journal#21 will be enough.

**Comment by eduardoj** [comment]
@jreidinger , @lslezak, done. Please re-review.

**Review by lslezak — APPROVED** [review]
LGTM, @jreidinger ?

**Review by jreidinger — APPROVED** [review]
LGTM

**Pull request merged**